### PR TITLE
Move getters to web worker

### LINKED ISSUE #3250: Move getters into web worker

### PR DESCRIPTION
**Pull request by adrake33**
Closes #3250. Unfortunately, this change did not fix the large number of threads, but I did notice that commenting out all code related to FlexSearch caused them to go away, so apparently there is something else related to FlexSearch that's still causing this. I filed #3271 to investigate this further.

### CODE REVIEW

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ export const handleSDKReadyEvent = () =>  (
   dispatch: ThunkDispatch<void, any, Action>
 ) => {
   // wire up events for sdk
+console.log("In handleSDKReadyEvent");
```
Comment (bthaile) [inline packages/augur-ui/src/modules/events/actions/log-handlers.ts]: Is this console log needed for debugging whe app is running?

Comment (adrake33) [reply packages/augur-ui/src/modules/events/actions/log-handlers.ts]: Oops, no I meant to remove this one and just missed it.

**Comment by bthaile** [comment]
Have you cranked up the UI to test that it doesnt blow up? Code looks fine

**Comment by adrake33** [comment]
The markets show up in the UI, but as I mentioned in the PR description, it doesn't fix the large number of threads. Still need to look into that more.

**Review by bthaile — APPROVED** [review]
Lgtm

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 📈
 To view it, be sure to [join Augur Project](https://dashboard.featurepeek.com/login) on FeaturePeek.

 This link will always point to this PR's latest changes: https://peek.run/l74hzze

**Pull request merged**